waspc/src/Wasp/AppSpec/App/Auth.hs

```diff
@@ -13,7 +13,8 @@ import Wasp.AppSpec.Entity (Entity)
 data Auth = Auth
   { userEntity :: Ref Entity,
     methods :: [AuthMethod],
-    onAuthFailedRedirectTo :: Maybe String
+    onAuthFailedRedirectTo :: String,
```

Comment:
I remember we had some discussion about this property seeming to be mandatory, but in practice, it isn't, right? Should we mention that here and maybe link to the appropriate GH issue if there is one?

Reply:
We agreed it should be mandatory, because we can't know where you want to go upon failure, which is why I made it just `String`, with no `Maybe`.

Is that ok or do you think it requires further discussion? When you mentioned comments, what should I write in them? What would the GH issue be about?

Reply:
all good, I mixed it up with onAuthSucceeded which is Maybe, so that is ok!